pkg/cluster/cluster_config.go

```diff
@@ -141,7 +141,7 @@ func GetRelease(ctx context.Context, cli client.Client) (Release, error) {
 		},
 	}
 	// Set platform
-	platform, err := GetPlatform(ctx, cli)
+	platform, err := getPlatform(ctx, cli)
```

Comment:
ignore my previous comments.

Reply:
No problem. Does the change make sense? Since Init() patch depends of it. It's ok for me to keep GetPlatform, but  with the Init it makes the code more clear since there is no that internal dependency you mentioned.

Reply:
sure, i get these name messed up, but it looks good to me